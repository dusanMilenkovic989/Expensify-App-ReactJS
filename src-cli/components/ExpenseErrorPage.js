import React from 'react'
import { Link } from 'react-router-dom'

const ExpenseErrorPage = () => (
    <div>
        <p>404 Page</p>
        <Link to="/dashboard">Go home</Link>
    </div>
)

export { ExpenseErrorPage as default }