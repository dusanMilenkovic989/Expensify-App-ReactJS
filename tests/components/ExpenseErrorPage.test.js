import React from 'react'
import { shallow } from 'enzyme'
import ExpenseErrorPage from '../../src-cli/components/ExpenseErrorPage'

test('Should render ExpenseErrorPage correctly', () => {
    const wrapper = shallow(<ExpenseErrorPage />)

    expect(wrapper).toMatchSnapshot()
})