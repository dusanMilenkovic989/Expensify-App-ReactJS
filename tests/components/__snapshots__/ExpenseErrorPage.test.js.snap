// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseErrorPage correctly 1`] = `
<div>
  <p>
    404 Page
  </p>
  <Link
    to="/dashboard"
  >
    Go home
  </Link>
</div>
`;
